RW-177 Preview

left23 opened the pull request:
- Hide view mode select element
- Style Back button so it's easily visible. Its style is similar to the "Skip to main content" link, and I need to test with proper user access to see if their placements conflict (Eg. Will the admin toolbar always be present when users can Preview?)

![Selection_058](https://user-images.githubusercontent.com/1835923/137235207-09331b3d-3d50-457f-9cd1-76292a5153f6.png)

![Selection_059](https://user-images.githubusercontent.com/1835923/137235226-c492b692-6442-4f69-8b69-45f5c6cda4e1.png)

orakili reviewed (commented):
@left23 Looks good, thanks! 2 things I noticed:

1. The preview for the terms (ex: country) is not styled. I think it needs an override of `form--term-preview-form-select.html.twig` like done for the node one.
2. The "back to content editing" is not sticky anymore. I thought it was useful when reviewing a long document. What do you think?

Also wondering if another color for the "preview" banner would not make it more obvious.

left23 commented:
@orakili Yes, good point about the sticky header. I've made it sticky now and added top padding to the `dialog-off-canvas-main-canvas` div so everything is still visible before the page is scrolled.

For now, I've made the header red `--cd-reliefweb-dark-red: #eb1405;` Let me know if you think another colour is better.

I've also added another template override for the term preview.
![Selection_067](https://user-images.githubusercontent.com/1835923/137409746-d5f0b54b-f56b-485e-af97-2ad7936eff08.png)
